FrontEnd/FrontEnd.Web/Components/Layout/NavMenu.razor

```diff
@@ -44,6 +51,7 @@
             var user = authState.User;
 
             _loggedIn = user.Identity is not null && user.Identity.IsAuthenticated;
+            _isAdmin = user.HasClaim(c => c is { Type: ClaimTypes.Role, Value: UserRoles.Admin });
```

Comment:
Does `user.IsInRole` help?

Reply:
Cool, thanks

Reply:
FYI you could even have used [AuthorizeView](https://learn.microsoft.com/en-us/aspnet/core/blazor/security/?view=aspnetcore-8.0#role-based-and-policy-based-authorization), but let's not care about  it now